### PR TITLE
Adding PrivateEndpoint CUJ notebook to notebooks/community

### PR DESCRIPTION
**Pull request by nayaknishant**
Adding official SDK notebook for `PrivateEndpoint` after Vertex AI 1.15 SDK release. Alphabetized/organized CODEOWNERS in official folder. 

If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [X] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [X] Follow the style and grammar rules outlined in the above notebook template.
- [X] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [X] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [X] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [X] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/community/sdk/sdk_private_endpoint.ipynb [hunk 1]
```diff
@@ -0,0 +1,1015 @@
+{
```
Comment (andrewferlitsch) [inline notebooks/community/sdk/sdk_private_endpoint.ipynb:1]: <p>Subnetwork -&gt; subnetwork</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (andrewferlitsch) [inline notebooks/community/sdk/sdk_private_endpoint.ipynb:1]: <p>load_ext tensorboard -&gt; if tensorboard not used in notebook, you should remove the extension</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Comment (andrewferlitsch) [inline notebooks/community/sdk/sdk_private_endpoint.ipynb:1]: <p>Use the preferred form for links to webdocs:</p><p> </p><p>Learn more about [title](link).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Comment (andrewferlitsch) [inline notebooks/community/sdk/sdk_private_endpoint.ipynb:1]: <p>delete_bucket should be set to False</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>

Comment (nayaknishant) [reply notebooks/community/sdk/sdk_private_endpoint.ipynb:1]: Fixed, removed the extension

Comment (nayaknishant) [reply notebooks/community/sdk/sdk_private_endpoint.ipynb:1]: Fixed

**Pull request merged**